src/ts/index.ts

```diff
@@ -162,6 +162,30 @@ function encodeSigningScheme(v: number, scheme: SigningScheme): number {
   }
 }
 
+/**
+ * Details representing how an order was executed.
+ */
+export interface TradeExecution {
+  /**
+   * The executed trade amount.
+   *
+   * How this amount is used by the settlement contract depends on the order
+   * flags:
+   * - Partially fillable sell orders: the amount of sell tokens to trade.
+   * - Partially fillable buy orders: the amount of buy tokens to trade.
+   * - Fill-or-kill orders: this value is ignored.
+   */
+  executedAmount: BigNumberish;
+  /**
+   * Optional fee discount to use in basis points (1/100th of 1%).
+   *
+   * If this value is `0`, then there is no discount and the full fee will be
+   * taken for the order. A value of `10000` is used to indicate a full
+   * discount, meaning no fees will be taken.
+   */
```

Comment:
This is not implemented in the contract yet, right? Since using basis points are kind of pointless, we could say that each unit is 1/(256²-1)-th of the fee amount, just to avoid the ugly extra bound check in the contract.

Reply:
It is not implemented. Personally I like basis points as its fairly standard.

> just to avoid the ugly extra bound check in the contract.

It would be something caught by `SafeMath` anyway.